gubernator/update_config.py

```diff
@@ -17,23 +17,36 @@
 """Updates the Gubernator configuration from the Prow configuration."""
 
 import argparse
+import os
 import yaml
 
-def main(prow_config, gubernator_config):
-    prow_data = yaml.load(open(prow_config))
+def main(prow_config, prow_job_config, gubernator_config):
+    configs = [prow_config]
```

Comment:
Side note: I think in the future maybe we should have something like `prow/cmd/config` be able to scoop up the configs and spit out a single json/yaml blob to be consumed by tools like this so they can just inspect the keys of a loaded config instead of trying to re-implement all of the loading behavior.